front-end/src/app/shared/components/contact-dialog/contact-dialog.component.ts

```diff
@@ -329,6 +329,13 @@ export class ContactDialogComponent extends DestroyerComponent implements OnInit
 
   updateContact(contact: Contact) {
     this.contact = contact;
+    if (this.contact.candidate_state && this.contact.candidate_district === '') {
+      this.candidateDistrictOptions = LabelUtils.getPrimeOptions(
+        LabelUtils.getCongressionalDistrictLabels(this.contact.candidate_state),
+      );
+      this.contact.candidate_district = '00';
+    }
+
```

Comment:
This is setting the candidate district to 00 for every Senate candidate, _buuuuuut_ it's immediately getting cleared out _somehow_.  Visually, I don't ever see a 00 in the district field for senators retrieved with Contact Lookup, but using my browser's debugger, the assignment on line #336 is getting run.

I'd recommend adding one further check to ensure that it's a House candidate, just to be safe.

Reply:
Oh good catch. Limited it to just be done if the type is House.

Reply:
Looks like we're just addressing the symptom here.  When we get an alaska from the api, we're actually receiving a district.  https://github.com/fecgov/fecfile-web-app/commit/cc709bd32d566017974ac0360a36dd3715f9852d seems to be overwriting it.  Can you look into https://github.com/fecgov/fecfile-web-app/issues/1118 and see what problem matt was trying to address by setting it to '' when the district is '00' so that we can handle that scenario and still pull in district from candidates